src/main/java/view/HomePageView.java

```diff
@@ -81,6 +93,13 @@ public HomePageView(HomepageController homepageController, PostController postCo
         // });
     }
 
+    // Seperate dialog hopefully thats okay
```

Comment:
So a separate dialog is fine I think, if we have time we can implement the Card Layout behavior.

What does everyone else think?